lib/common.py

```diff
@@ -608,7 +632,11 @@ def pbd_uuids(self):
     def unplug_pbds(self):
         print("Unplug PBDs")
         for pbd_uuid in self.pbd_uuids():
-            self.pool.master.xe('pbd-unplug', {'uuid': pbd_uuid})
+            # We must be sure to execute correctly "unplug" on unplugged VDIs without error.
+            try:
+                self.pool.master.xe('pbd-unplug', {'uuid': pbd_uuid})
+            except Exception as e:
+                print('Ignore exception during PBD unplug: {}'.format(e))
```

Comment:
I suppose it would be better if we catch a more precise exception like "already unplugged". For a better compromise, I can add a `force` parameter to this helper to ignore this exception.

Reply:
agreed to both